test/browser.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const assert = require('assert');
+const wd = require('wd');
+const util = require('./util');
+const abab = require('..').btoa;
+const atob = abab.atob;
+const btoa = abab.btoa;
+
+const browsers = ['chrome', 'firefox', 'internet explorer'];
+
+if (process.env.SAUCE_USER) {
+  const SAUCE_USER = process.env.SAUCE_USER;
+  const SAUCE_KEY = process.env.SAUCE_KEY;
+
+  describe('browser compatibility', function () {
+    browsers.forEach(function (browserName) {
+      describe(browserName, function () {
+        var browser;
+        before(function () {
+          this.timeout(10000);
+          return browser = wd.promiseChainRemote('ondemand.saucelabs.com', 80, SAUCE_USER, SAUCE_KEY)
+            .init({browserName: browserName})
+            .get('about:blank');
+        });
+        after(function () {
+          return browser.quit();
+        });
+
+        describe('atob', function () {
+          const data = require('./fixtures/atob');
+          const cases = data.cases;
+          const answers = data.answers;
+          cases.forEach(function (input, index) {
+            const expected = util.getAnswer(answers, index);
+            const inputDescriptor = util.stripChars(input);
+            const expectedDescriptor = util.stripChars(expected);
+
+            it(`correctly converts ${inputDescriptor} into ${expectedDescriptor}`, function () {
+              return browser.eval(`atob('${input}')`).then(function (result) {
```

Comment:
I think the `eval`'d statement needs to be changed - for cases that don't throw a `TypeError`, it works, but for cases when it does, it fails.